ordr-d/root/helx-startup/10-user.sh

```diff
@@ -2,58 +2,58 @@
 
 set -eoux pipefail
 
-HELX_GROUP_NAME=${HELX_GROUP_NAME-"helx"}
```

Comment:
Why are we commenting out most of this file?

Reply:
The current image `helx-notebook:ordrd-v0.0.5` has these same comments in this script, but those changes never seem to have gotten committed. I'm trying to figure out who built that image so that I can get some info about why it was done, but all I can say for now is that the script causes issues with user identity in ordrd